mappings/net/minecraft/item/GoatHornItem.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_7430 net/minecraft/item/GoatHornItem
+	FIELD field_39052 SOUND_VARIANT_KEY Ljava/lang/String;
+	METHOD method_43558 getStackForVariant (I)Lnet/minecraft/class_1799;
+		ARG 0 soundVariant
+	METHOD method_43559 getRandomVariant (Lnet/minecraft/class_5819;Z)I
```

Comment:
either all use SoundVariant suffix or all just Variant suffix, a mix is confusing